src/cryptenroll/cryptenroll-pkcs11.c

```diff
@@ -18,8 +17,8 @@ int enroll_pkcs11(
         _cleanup_(erase_and_freep) char *base64_encoded = NULL;
         _cleanup_(json_variant_unrefp) JsonVariant *v = NULL;
         _cleanup_free_ char *keyslot_as_string = NULL;
-        size_t decrypted_key_size, savedata_size;
-        _cleanup_free_ void *savedata = NULL;
+        size_t decrypted_key_size, saved_key_size;
+        _cleanup_free_ void *saved_key = NULL;
```

Comment:
you just renamed this variable in the previous patch of this PR. We generally want that patches in the same PR do not "fix" changes made in the very same PR earlier. it's fine to rename stuff if there's a good reason to, but then do it only once and stick to it through the rest of the PR.